ltr/concourse/Dockerfile

```diff
@@ -0,0 +1,14 @@
+# See https://www.tensorflow.org/install/pip?lang=python3#package-location for supported Python versions
```

Comment:
I think it varies, and it would definitely be good to get someone more familiar with Concourse to have a look at this PR too.  We could put everything in the yaml file, it would just get a bit verbose.

Reply:
I've refactored the pipeline a bit to be more concourse-standard: build outputs are stored in an s3 bucket ([RE provision an s3 bucket for every concourse team](https://reliability-engineering.cloudapps.digital/continuous-deployment.html#services)), and jobs look at that state, rather than figuring out what the latest state is by making AWS API calls.